racetrack_client/racetrack_client/main.py

```diff
@@ -69,10 +70,17 @@ def _print_help(_: argparse.Namespace):
     parser_delete = subparsers.add_parser('delete', help='Delete fatman instance')
     parser_delete.add_argument('workdir', default='.', nargs='?', help='directory with fatman.yaml manifest')
     parser_delete.add_argument('racetrack_url', default='', nargs='?', help='URL to Racetrack server or alias name')
-    parser_delete.add_argument('--version', nargs='?', type=str,
-                               help='fatman version to delete')
+    parser_delete.add_argument('--version', nargs='?', type=str, help='fatman version to delete')
     parser_delete.set_defaults(func=_delete_fatman)
 
+    # racetrack move
+    parser_move = subparsers.add_parser('move', help='Move fatman from one infrastructure target to another fatman instance')
+    parser_move.add_argument('--remote', type=str, help='URL to Racetrack server or alias name')
```

Comment:
```suggestion
    parser_move.add_argument('--remote', type=str, help='Racetrack server\'s URL or alias name')
```

Nitpick: this feels like more "natural" English to me.

Reply:
Good point